web/consts/consts.go

```diff
@@ -0,0 +1,82 @@
+// Package consts contains constant strings used on HTTP handlers
+package consts
+
+import (
+	"strings"
+	"unicode"
+)
+
+const (
+	// Accept is the canonical header name used for negotiating a MIME Type
+	// for the content of the Request response
+	Accept = "Accept"
+
+	// AcceptEncoding is the canonical name given to the header used
+	// to indicate compression options
+	AcceptEncoding = "Accept-Encoding"
+
+	// Allowed is the canonical header used to indicate the supported
+	// Methods.
+	Allowed = "Allowed"
+
+	// ContentEncoding is the canonical Content-Encoding header
+	ContentEncoding = "Content-Encoding"
+	// ContentLength is the canonical Content-Length header
+	ContentLength = "Content-Length"
+	// ContentType is the canonical Content-Type header
+	ContentType = "Content-Type"
+	// ETag is the canonical ETag header
+	ETag = "Etag"
+
+	// Location is the canonical name given to the header used
+	// to indicate a redirection.
+	Location = "Location"
+)
+
+const (
+	// GET represents the HTTP GET Method.
+	GET = "GET"
+	// HEAD represents the HTTP HEAD Method.
+	HEAD = "HEAD"
+	// POST represents the HTTP POST Method.
+	POST = "POST"
+	// PUT represents the HTTP PUT Method.
+	PUT = "PUT"
+	// DELETE represents the HTTP DELETE Method.
+	DELETE = "DELETE"
+	// OPTIONS represents the HTTP OPTIONS Method.
+	OPTIONS = "OPTIONS"
+	// PATCH represents the HTTP PATCH Method.
+	PATCH = "PATCH"
+)
+
+const (
+	// JSON is the standard Media Type for JSON content.
+	JSON = "application/json; charset=utf-8"
+	// HTML is the standard Media Type for HTML content.
+	HTML = "text/html; charset=utf-8"
+	// TXT is the standard Media Type for plain text content.
+	TXT = "text/plain; charset=utf-8"
```

Comment:
^^ this is the default MIME for everything text based (not html as one might guess). I miss the other default MIME type (the one or everything NOT text which is `application/octet-stream`)

Reply:
added as `BIN` now